### PR TITLE
Support for colors

### LINKED ISSUE #25: Consider automatically using different colors on black/white backgrounds

### PR DESCRIPTION
**Pull request by AlexGhiondea**
Fixes #25 

/cc @safern

### CODE REVIEW

File: src/Parser.HelpColors.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                }
+
+                return s_helpColors;
+            }
+
+            private static IColors GetDefault(ConsoleColor backgroundColor)
+            {
+                switch (backgroundColor)
+                {
+                    case ConsoleColor.Black:
```
Comment (safern) [inline src/Parser.HelpColors.cs]: Couldn't you write this with just one return per color kind and then use the case cascade? i.e:

```cs
case ConsoleColor.Black:
case ConsoleColor.Blue:
case ConsoleColor.DarkBlue:
    return new DarkBackgroundColors();
...
```

File: test/ColorizerWriterObject/TestWriter.cs [hunk 2]
```diff
@@ -15,6 +15,7 @@ public ConsoleColor ForegroundColor
         {
             get
             {
+
```
Comment (safern) [inline test/ColorizerWriterObject/TestWriter.cs]: Nit: extra line?

File: src/Parser.HelpColors.cs (hunk 1 above)
Comment (AlexGhiondea) [reply src/Parser.HelpColors.cs]: Good point! I will do that! Originally I think I had different colors for each one :).

**Comment by AlexGhiondea** [comment]
Thanks for the review @safern

**Pull request merged**